src/pybind/mgr/dashboard_v2/controllers/block_pool.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+import cherrypy
+import rbd
+
+from ..tools import ApiController, AuthRequired, BaseController, RemoteViewCache
+
+
+@ApiController('block_pool')
+@AuthRequired()
+class Host(BaseController):
+    def __init__(self):
+        # Stateful instances of RbdLs, hold cached results.  Key to dict
+        # is pool name.
+        self.rbd_ls = {}
+
+    @cherrypy.expose
+    @cherrypy.tools.json_out()
+    def rbd_pool_data(self, pool_name):
+        rbd_ls = self.rbd_ls.get(pool_name, None)
+        if rbd_ls is None:
+            rbd_ls = RbdLs(self.mgr, pool_name)
+            self.rbd_ls[pool_name] = rbd_ls
+
+        status, value = rbd_ls.get()
+        return {'status': status, 'value': value}
+
+
+class RbdLs(RemoteViewCache):
+    def __init__(self, module_inst, pool):
+        super(RbdLs, self).__init__(module_inst)
+
+        self.pool = pool
+
+        self.ioctx = None
+        self.rbd = None
+
+    def _init(self):
+        self.log.debug("Constructing IOCtx")
+        self.ioctx = self._module.rados.open_ioctx(self.pool)
+
+        self.log.debug("Constructing RBD")
+        self.rbd = rbd.RBD()
+
+    def _get(self):
+        self.log.debug("rbd.list")
+        names = self.rbd.list(self.ioctx)
+        result = []
+        for name in names:
+            i = rbd.Image(self.ioctx, name)
+            stat = i.stat()
+            stat['name'] = name
+            features = i.features()
+            stat['features'] = features
+            stat['features_name'] = self._format_bitmask(features)
+
+            try:
+                parent_info = i.parent_info()
+                parent = "{}@{}".format(parent_info[0], parent_info[1])
+                if parent_info[0] != self.pool:
+                    parent = "{}/{}".format(parent_info[0], parent)
+                stat['parent'] = parent
+            except rbd.ImageNotFound:
+                pass
+            result.append(stat)
+        return result
+
+    def _format_bitmask(self, features):
+        names = ""
+        RBD_FEATURES_NAME_MAPPING = {
+            rbd.RBD_FEATURE_LAYERING: "layering",
+            rbd.RBD_FEATURE_STRIPINGV2: "striping",
```

Comment:
I'd keep the name `stripingv2` here, instead of inventing a new name.

Reply:
My hesitation in changing this name is due to the fact that `striping` is the name used in the legacy dashboard (v1). 

Maybe there is a reason why this name had been chosen...